wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -27,9 +27,13 @@ Images or visual elements containing any of the following are **not allowed**:
 - **significant sexual innuendo**
 - **sexual posturing or solicitation**
 - **erotic content or graphic displays of sexuality**
-- **examples of drug use, preparation, or identification with illicit drug subculture**
+- **examples of drug use, preparation, or identification with illicit drug subculture**¹
 - **depictions of deliberately inflammatory political, cultural, religious, or social content**
 
+In addition, images or visual elements that could be reasonably appraised as being of poor quality (in either objective image quality or subjective composition/creation) may also be prevented from being used in a beatmaps in cases where they are not directly related to any relevant subject matter.
+
+*¹*: This may be subject to exceptions depending on the nature of the drug, its usage, and its subculture, particularly in cases of drugs experiencing rapid worldwide decriminalization and regulation like marijuana. Requesting specific exceptions to this rule should be undertaken by contacting the osu!team at [support@ppy.sh](mailto:support@ppy.sh).
```

Comment:
```suggestion
*¹*: This may be subject to exceptions depending on the nature of the drug, its usage, and its subculture, particularly in cases of drugs experiencing rapid worldwide decriminalization and regulation like marijuana. Requesting specific exceptions to this rule should be undertaken by contacting the [osu!team](/wiki/People/The_Team) at [support@ppy.sh](mailto:support@ppy.sh).
```